system_setup/server/controllers/configure.py

```diff
@@ -58,8 +58,14 @@ def __locale_gen_cmd(self) -> Tuple[str, bool]:
         cmdFile = os.path.join(self.model.root, cmd)
         shutil.copy(os.path.join("/", cmd), cmdFile)
         # Supply LC_* definition files to avoid complains from localedef.
-        shutil.copytree("/usr/lib/locale/C.UTF-8/", outDir,
-                        dirs_exist_ok=True)
+        candidateSourceDirs = ["/usr/lib/locale/C.UTF-8/",
+                               "/usr/lib/locale/C.utf8/"]
+        sourceDirs = [d for d in candidateSourceDirs if os.path.exists(d)]
+        if len(sourceDirs) == 0 or sourceDirs is None:
```

Comment:
The "or sourceDirs is None" part should go, not only is there no way for sourceDirs to be None, len(sourceDirs) would have crashed before this is evaluated if was.

Reply:
Got it. Removed the unnecessary check.